src/modules/layout/components/header/user-menu.tsx

```diff
@@ -44,16 +72,13 @@ export function UserMenu() {
         }}
         sx={{ mt: 1 }}
       >
-        <MenuItem>
-          <Link href="/auth/login" sx={{ width: 150 }}>
-            Log in
-          </Link>
-        </MenuItem>
-        <MenuItem onClick={handleSignUpClick}>
-          <Link href="/auth/register" sx={{ width: 150 }}>
-            Sign up
-          </Link>
-        </MenuItem>
+        {menuItems.map((item, index) => (
+          <MenuItem key={index} onClick={item.action}>
+            <Link href="" sx={{ width: 150 }}>
```

Comment:
Could we have the links be the links? For example:

Login and register button are actually links 
Logout is button since it does some extra stuff there